Loading fixtures doesn't work (because forum-reserved-names.js isn't ignored)

Following [the instructions to load fixtures](https://github.com/DemocracyOS/democracyos/wiki/Load-fixtures) caused the following to happen (with `<...>` in place of private-looking values):

    >>> NODE_PATH=. node ./bin/dos-db load tag ./lib/fixtures/tags.json
    { [Error: Cannot find module '../build/Release/bson'] code: 'MODULE_NOT_FOUND' }
    js-bson: Failed to load c++ bson extension, using pure JS version
    Connecting to mongodb://<...>/DemocracyOS-dev
    Register mongoose models.
    <...>/democracyos/node_modules/mongodb/lib/mongodb/db.js:299
              throw err;
              ^

    TypeError: require(...) is not a function
        at <...>/democracyos/bin/dos-db:141:43
        at Array.forEach (native)
        at models (<...>/democracyos/bin/dos-db:139:18)
        at <...>/democracyos/bin/dos-db:44:7
        at open (<...>/democracyos/node_modules/mongoose/lib/connection.js:452:17)
        at NativeConnection.Connection.onOpen (<...>/democracyos/node_modules/mongoose/lib/connection.js:462:5)
        at <...>/democracyos/node_modules/mongoose/lib/connection.js:421:10
        at <...>/democracyos/node_modules/mongoose/lib/drivers/node-mongodb-native/connection.js:59:5
        at <...>/democracyos/node_modules/mongodb/lib/mongodb/db.js:296:11
        at nextTickCallbackWith0Args (node.js:456:9)

The reason for this appears to be that `except` in line 137 of `./bin/dos-db` doesn't include `forum-reserved-names.js`, whose export is not a function. Adding `forum-reserved-names.js` to the list causes everything to work as expected.

--- a/bin/dos-db
+++ b/bin/dos-db
@@ -134,7 +134,7 @@ function connect (fn) {
  */
 function models(fn) {
   write("Register mongoose models.");
-  var except = ['index.js'];
+  var except = ['index.js', 'forum-reserved-names.js'];
   var files = dir(resolve('./lib/models/'));
   files && files.forEach(function(f) {
     if (~except.indexOf(f)) return;

